src/docker/10-debianconsole/.prebuild.sh

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
 set -e
 
+# FIXME Remove this when Docker supplies multi-arch base images
+. .$ARCH.base-images
+docker pull $DEBIAN_BASE_IMAGE
```

Comment:
Ok,  changed. I had to do the check a bit differently because `docker inspect` returns an error if the image does not exist, which does not work with `set -e`.